### ISSUE TITLE
Working on graph produces a code that does not parse

### ISSUE THREAD

**Issue by wdanilo** [luna/luna-studio#1123]
It's critical. It makes working on some functions impossible in the visual environment.

### Steps to Reproduce

1. tab -> `def foo:1`
2. enter foo node
3. remove the `1` node
4. create nodes `12` and `19`

### Expected Result

We should get the code:

```haskell
import Std.Base

def main:
    def foo:
        number1 = 12
        number2 = 19
        None
    None
```

### Actual Result

We get a code which does not parse correctly:

```haskell
import Std.Base

def main:
    def foo:number1 = 12
            number2 = 19
            None
    None
```

### Luna Version
1.4.101.6

**Comment by mikusp** [comment]
Is it correct code that does not parse or incorrect code? If incorrect, what should be the behaviour if we enter `def foo:1` and add a new node? Should it be added on the same indentation level as `1` or the whole block should be moved?

**Comment by piotrMocz** [comment]
@mikusp I'd move the whole block, or even better, never create this inline form in the text representation.
This way if I typed `def foo: 1`, I'd get
```python
def foo:
    1
```
And any subsequent lines would fit nicely. I.e. let's just treat this inline form as a syntactic sugar of sorts?

**Comment by wdanilo** [comment]
Yes, we need to move the whole block down. Why I'm assigned here and not just mentioned?
